apps/web/src/design-system/name-input/NameInput.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ChangeEvent } from 'react';
+import { TextInput as MantineTextInput } from '@mantine/core';
+import { SpacingProps } from '../shared/spacing.props';
+import { nameInputStyles } from './nameInputStyles';
+
+interface INameInputProps extends SpacingProps {
+  placeholder?: string;
+  required?: boolean;
+  disabled?: boolean;
+  value?: string;
+  onChange?: (event: ChangeEvent<HTMLInputElement>) => void;
+}
+
+/**
+ * NameInput component
+ *
+ */
+export const NameInput = React.forwardRef<HTMLInputElement, INameInputProps>(
```

Comment:
Created a component for the input titles, we use it in workflow name, step name and now in instance name.